packages/ui/libs/ui/core/src/components/icons/FilterIcon.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useOriginConfiguration } from '../../utils/configuration';
+
+export function FilterIcon() {
+  const { styleConfig } = useOriginConfiguration();
```

Comment:
We are not using this anymore in core package. We should use colors from MUI theme